src/log.js

```diff
@@ -67,12 +136,13 @@ numericLogLevel._loglevelMap = _loglevelMap;
  * If any error is encountered a less informative string than a full
  * inspect is returned and the error is logged using `err()`.
  *
+ * @function
  * @param {*} what The object to inspect
  * @param {Object} opts Options will be passed through to inspect.
  *   Note that these may be ignored if there is an error during inspect().
  */
-const tryInspect = (what, opts) => {
-  opts = { depth: null, breakLength: Infinity, ...opts };
+const tryInspect = (what, opts_ = {}) => {
```

Comment:
I would keep the argument _nice_ (so it doesn't influence the jsdoc) and have the `opts_` in the function body.

Reply:
Ok…